os/board/common/Kconfig

```diff
@@ -54,3 +54,53 @@ config FLASH_PART_NAME
 
 endif # FLASH_PARTITION
 endif # ARCH_USE_FLASH && (NFILE_DESCRIPTORS != 0)
+
+config ARCH_USE_EXTERNAL_FLASH
+	bool
+	default n
+	---help---
+		This shows that user enables External FLASH to use.
```

Comment:
How can we enable this config? Could you add description?

Reply:
Because you remove the prompt, user can't control this config.
This config should be selected by chip config. That's why I request removing the prompt.